composer.json

```diff
@@ -19,6 +19,9 @@
         "symfony/browser-kit": "~2|~3",
         "symfony/process": "~2|~3"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.0"
```

Comment:
Just a thought - you can add [php-cs-fixer](https://github.com/google/google-auth-library-php/blob/master/composer.json#L18) to composer.json and then call it from `vendor/bin` like we do [here](https://github.com/google/google-auth-library-php/blob/master/.travis.yml#L26)